spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/File.scala

```diff
@@ -220,15 +235,24 @@ private[bigdl] class FileReader(fileName: String) {
   private val conf = File.getConfiguration(fileName)
   private val path = new Path(fileName)
   private val fs: FileSystem = path.getFileSystem(conf)
+  private[bigdl] val httpPrefix: String = "http"
```

Comment:
what about https?

Reply:
why private bigdl?

Reply:
Because I use startwith to judge whether it is http and 'http' and 'https' both start with 'http', so it will work for both of them.

Reply:
what about if user has a typo, something like httpf?